source/en/api_nearest_neighbor.rst

```diff
@@ -32,6 +32,21 @@ We show each field below:
    Specify parameters for the algorithm.
    Its format differs for each ``method``.
 
+   common
+     :threads:
+        The number of threads execute random_projection and search.
+        The bigger it is, query latency becomes smaller because of data divided and excuted by multi thread in parallel.
+        This option added since 0.9.0. Previously 0.8.9, it worked with 1 thread.
```

Comment:
This option has been added since 0.9.0. Single thread is used in before versions.